mod_reforged/hooks/config/character.nut

```diff
@@ -25,6 +25,9 @@ local getClone = ::Const.CharacterProperties.getClone;
 ::Const.CharacterProperties.OffensiveReachIgnore <- 0;
 ::Const.CharacterProperties.BonusPerReachAdvantage <- 0;
 
+::Const.CharacterProperties.IsImmuneIsOnToEnterPushback <- false;	// Will an enemy spearwall push me back to my original tile when they hit the AOO?
```

Comment:
typo extra `Is` after `IsImmune`

Reply:
It was not actually a typo. `IsOnToEnter` is the name of the variable used to determine whether the target entity entered my tile from the outside or tries to leave my tile.
But even still I don't like my name.

How about `IsImmuneToOnEnterPushback`?
So a **To** instead of the **Is**